tests/run.bats

```diff
@@ -574,7 +574,7 @@ function configure_and_check_user() {
 	skip_if_no_runtime
 
 	_prefetch alpine
-	run_buildah from --quiet --pull=false $WITH_POLICY_JSON alpine
+	run_buildah from --quiet --pull=ifmissing $WITH_POLICY_JSON alpine
```

Comment:
This one shouldn't need changing - the named image is prefetched.  Does it start to fail without this change?